include/cucumber-cpp/internal/ContextManager.hpp

```diff
@@ -16,7 +18,7 @@ namespace internal {
 
 typedef std::vector<shared_ptr<void> > contexts_type;
 
-class ContextManager {
+class CUCUMBER_CPP_EXPORT ContextManager {
```

Comment:
I think it's possible to export only the member function `purgeContexts` and add a non-template overload for `addContext` and export that too.

But this can also be done after this PR.